### ISSUE TITLE
No named parameter with the name 'size'. print_commands

### ISSUE THREAD

**Issue by leandroviegas** [Eddayy/flutter_star_prnt#53]
I just updated the flutter to the new version and I'm getting this error:

/C:/Users/Leandr/AppData/Local/Pub/Cache/hosted/pub.dev/flutter_star_prnt-2.4.0/lib/src/print_commands.dart:205:9: Error: No named parameter with the name 'size'.
        size: logicalSize,
        ^^^^
/C:/flutter/packages/flutter/lib/src/rendering/view.dart:28:9: Context: Found this candidate, but the arguments don't match.
  const ViewConfiguration({
        ^^^^^^^^^^^^^^^^^
Target kernel_snapshot failed: Exception

FAILURE: Build failed with an exception.

* Where:
Script 'C:\flutter\packages\flutter_tools\gradle\src\main\groovy\flutter.groovy' line: 1629

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\flutter\bin\flutter.bat'' finished with non-zero exit value 1

**Comment by nodahikaru** [comment]
I am getting same error. Help wanted

**Comment by salarazad** [comment]
I'm having the same issue now after Flutter latest update , please any help

**Comment by khp53** [comment]
You guys can use my fork:
```
flutter_star_prnt:
    git:
      url: https://github.com/khp53/flutter_star_prnt_mod.git
      ref: master
```

**Comment by salarazad** [comment]
> You guys can use my fork:
> 
> ```
> flutter_star_prnt:
>     git:
>       url: https://github.com/khp53/flutter_star_prnt_mod.git
>       ref: master
> ```

It seems to be working fine with me... thank you for this fork!

**Comment by luoqiang110** [comment]
Why hasn't the author fixed this issue yet?

**Comment by naposhens** [comment]
> You guys can use my fork:
> 
> ```
> flutter_star_prnt:
>     git:
>       url: https://github.com/khp53/flutter_star_prnt_mod.git
>       ref: master
> ```

still same got error

**Comment by khp53** [comment]
@naposhens can you paste the error log here?

**Comment by Naposhen** [comment]
@khp53 
../../.pub-cache/hosted/pub.dev/flutter_star_prnt-2.4.1/lib/src/print_commands.dart:207:9: Error: No named parameter with the name 'Size'.
        Size : logicalSize,
        ^^^^
../../development/flutter/packages/flutter/lib/src/rendering/view.dart:28:9: Context: Found this candidate, but the arguments don't match.
  const ViewConfiguration({
        ^^^^^^^^^^^^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command '/Users/shen/development/flutter/bin/flutter'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 33s
Error: Gradle task assembleDebug failed with exit code 1

**Comment by naposhens** [comment]
just paste in pubspec.lock right?

**Comment by naposhens** [comment]
> > You guys can use my fork:
> > ```
> > flutter_star_prnt:
> >     git:
> >       url: https://github.com/khp53/flutter_star_prnt_mod.git
> >       ref: master
> > ```
> 
> It seems to be working fine with me... thank you for this fork!

@salarazad add in pubspec.lock, right?

**Comment by salarazad** [comment]
> > > You guys can use my fork:
> > > ```
> > > flutter_star_prnt:
> > >     git:
> > >       url: https://github.com/khp53/flutter_star_prnt_mod.git
> > >       ref: master
> > > ```
> > 
> > 
> > It seems to be working fine with me... thank you for this fork!
> 
> @salarazad add in pubspec.lock, right?

exactly, I commented the original package dependency and added this fork below it:
`# flutter_star_prnt: ^2.4.1`
```
flutter_star_prnt:
    git:
      url: https://github.com/khp53/flutter_star_prnt_mod.git
      ref: master
```
try also to do 'flutter clean' and then 'flutter pub get' to see if it helps.

**Comment by Naposhen** [comment]
@salarazad Failed parsing lock file:

Error on line 543, column 7 of pubspec.lock: Expected a `version` entry.
    ╷
543 │ ┌       git:
544 │ │         url: https://github.com/khp53/flutter_star_prnt_mod.git
545 │ │         ref: master
546 │ │         version: "2.4.1"
547 │ │   flutter_switch:
    │ └──^
I got this error

**Comment by Naposhen** [comment]
> @naposhens can you paste the error log here?
Failed parsing lock file:

Error on line 543, column 7 of pubspec.lock: Expected a version entry.
╷
543 │ ┌ git:
544 │ │ url: https://github.com/khp53/flutter_star_prnt_mod.git
545 │ │ ref: master
546 │ │ 
547 │ │ flutter_switch:
│ └──^

**Comment by khp53** [comment]
@naposhens Make sure the indentation is right. And comment out the original dependency call from pubspec.yml file. Do a flutter clean and pub get.

**Comment by khp53** [comment]
```
flutter_star_prnt:
    git:
      url: https://github.com/khp53/flutter_star_prnt_mod.git
      ref: master
```

**Comment by salarazad** [comment]
check the line 543 in your pubspec.lock maybe it is still referring to the old original package.
check the url part for the flutter_star_prnt - it should be (url: "https://github.com/khp53/flutter_star_prnt_mod.git") and not (url: "https://pub.dev").
If it is still referring to the old package then delete flutter_star_prnt part and rebuild your app , or delete pubspec.lock file completely and do th 'flutter clean' & 'flutter pub get' and the pubspec.lock should be regenerated again with the correct references.

**Comment by mrnicon** [comment]
@naposhens , @khp53, @salarazad is it possible to make a new version/fork available on pub.dev? I use FlutterFlow and don't have the possibility to point directly to the github repo, only to pub.dev packages. 
Also it will make it easier to have the fixed to everyone.

**Comment by khp53** [comment]
@mrnicon Done, here is the package: https://pub.dev/packages/flutter_star_prnt_plus

**Comment by khp53** [comment]
I will maintain the package from now on, if you guys find any issues list them on the github repo attached to the new package. Also, feel free to contribute and create PRs.

**Comment by mrnicon** [comment]
@khp53, thank you so much, this made my day.
